content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -7,29 +7,30 @@ weight: 30
 ---
 
 {{% capture overview %}}
-This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
+This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this example, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
 
-Deploying stateful distributed applications, like Cassandra, within a clustered environment can be challenging. StatefulSets greatly simplify this process. Please read about [StatefulSets](/docs/concepts/workloads/controllers/statefulset/)  for more information about the features used in this tutorial.
+It can be challenging to deploy stateful distributed applications within a clustered environment. StatefulSets greatly simplify this process. See the [StatefulSet](/docs/concepts/workloads/controllers/statefulset/) documentation for more information on the features used in this tutorial.
 
-**Cassandra Docker**
+**Cassandra on Docker**
```

Comment:
TL,DR: Total nit, pay it no mind.

Just thinking idly: what's the proper prepositional phrase to describe the relationship of a container to its contents? Is Cassandra _on_ Docker? Is it _in_ Docker? Is it a conjunction instead&mdash;Cassandra _and_ Docker?

@Bradamant3, do you have thoughts? It's not worth holding up this PR, but I'm genuinely curious if you've encountered this usage scenario elsewhere.